view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -304,8 +303,7 @@ define(
                                     'screen_color_depth': browserInfo.colorDepth,
                                     'screen_width': browserInfo.screenWidth,
                                     'screen_height': browserInfo.screenHeight,
-                                    'timezone_offset': browserInfo.timeZoneOffset,
-                                    'language': browserInfo.language
```

Comment:
we use this line of code with the browserinfo

Reply:
hmm seems to be a rebase issue will revert thanks.

Reply:
done